### PR TITLE
Add Logger

### LINKED ISSUE #862: Introduce logger
Introduce a logger to dynamically turn on/off logging

- Add IDE custom property `org.digma.plugin.enable.JCEFLogging: boolean` to toggle the logging (Plugin side)
- Add global environment `isLoggingEnabled: boolean` JCEF variable (Plugin side)
- Handle environment variable (UI Side) - 1h
- Introduce Logger library to dynamically set log level (on/off) (UI Side) - 1h

### PR DESCRIPTION
**Pull request by kshmidt-digma**
Closes #862

### CODE REVIEW

File: src/logger.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { Logger } from "./logging/Logger";
+import { LOG_LEVEL } from "./logging/types";
+
+export const logger = new Logger(
+  window.isLoggingEnabled === true ? LOG_LEVEL.DEBUG : LOG_LEVEL.NONE
```
Comment (opoliarush) [inline src/logger.ts:5]: mb instead of having `isLoggingEnabled` just set logLevel?

File: src/logging/Logger.ts [hunk 2]
```diff
@@ -0,0 +1,94 @@
+import { format } from "date-fns";
+import { LOG_LEVEL } from "./types";
+
+export class Logger {
+  private logLevel: number;
+  private showTimeStamp: boolean;
+  private showLogLevel: boolean;
+
+  public constructor(
+    logLevel: LOG_LEVEL,
```
Comment (opoliarush) [inline src/logging/Logger.ts:10]: SG: rename something like this `minLoglevelThreshold`

File: src/logger.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import { Logger } from "./logging/Logger";
```
Comment (opoliarush) [inline src/logger.ts:1]: mb, create this as index.ts file inside logging folder ?

File: src/logger.ts (hunk 1 above)
Comment (kshmidt-digma) [reply src/logger.ts]: Postponed to discuss the changes with the team

**Review by opoliarush — APPROVED** [review]

**Pull request merged**